VideoLocker/src/main/java/org/edx/mobile/loader/CoursesAsyncLoader.java

```diff
@@ -52,6 +45,18 @@ public CoursesAsyncLoader(Context context) {
         return result;
     }
 
+    private List<EnrolledCoursesResponse> getEnrolledCourses() {
+        List<EnrolledCoursesResponse> ret = null;
+
+        try {
+            ret = environment.getServiceManager().getEnrolledCourses(false);
```

Comment:
Yeah I had the same thing in mind, when I saw the call being reverted to the older version.

I myself don't have much experience in dealing with a cache using Retrofit. Maybe @1zaman or @bguertin can help us in providing a possibly better solution that gives us a cache alternative while using Retrofit.